pubsub/hazelcast/metadata.go

```diff
@@ -0,0 +1,5 @@
+package hazelcast
+
+type metadata struct {
+	hazelcastServers string
```

Comment:
How is authentication done in hazelcast? The other components have some authentication mechanism but here there is only a list of servers.

Reply:
Same as in the existing Hazelcast integration for state. Here https://github.com/dapr/components-contrib/blob/master/state/hazelcast/hazelcast.go#L48 we use default authentication credentials.